k8s/templates/frontend-pgbouncer.yaml

```diff
@@ -10,12 +10,16 @@ data:
   # Aron: I configured pgbouncer based on the settings
   # recommended by this article:
   # https://medium.com/futuretech-industries/postgres-raffle-10k-connections-35-mo-part-two-b4c2e0c86e37
-  POOL_MODE: "transaction"
+  # https://docs.djangoproject.com/en/1.11/ref/databases/#transaction-pooling-and-server-side-cursors
+  POOL_MODE: "session"
   SERVER_RESET_QUERY: "DISCARD ALL"
   # the custom listen backlog doesn't matter until we increase the nodes' somaxconn
   LISTEN_BACKLOG: "16384"
+  MAX_CLIENT_CONN: !!str {{ .Values.studioApp.pgbouncer.max_client_conn | default 100 }}
+  # set min pool size to the default, so the connections are held in pool if no activity
+  MIN_POOL_SIZE: !!str {{ .Values.studioApp.pgbouncer.min_pool_size | default .Values.studioApp.pgbouncer.pool_size | default 10 }}
```

Comment:
Will this double default work?

Reply:
I think it should work, as `default` is a function that checks if the given argument has been set, but I haven't tried two defaults before